tasks/run_cd4pe_job.rb

```diff
@@ -29,6 +29,7 @@ class GZipHelper
   # static class to decompress tar.gz files
   TAR_LONGLINK = '././@LongLink'.freeze
   SYMLINK_SYMBOL = '2'.freeze
+  PAX_HEADER = 'x'.freeze
```

Comment:
It sounds like it is also possible to have a pax header with a 'g' typeflag (which affects all files in the archive, whereas 'x' just affects the next file). See https://pubs.opengroup.org/onlinepubs/9699919799/utilities/pax.html#tag_20_92_13_02. Do we need to account for that too?

Reply:
We haven't seen that in the example tar file from the customer so I think we are ok to proceed with the change as is until there is evidence that a 'g' type flag is causing an issue.